manager.go

```diff
@@ -0,0 +1,220 @@
+package sandbox
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"net/http"
+
+	"github.com/bmizerany/pat"
+	"github.com/dchest/uniuri"
+	"gopkg.in/vinxi/layer.v0"
+	"gopkg.in/vinxi/vinxi.v0"
+)
+
+type Layer struct {
+	layer *layer.Layer
+}
+
+type Options struct {
+	Optional bool
+}
+
+type Rule interface {
+	ID() string
+	Name() string
+	Description() string
+	// Options() Options
+	JSONConfig() string
+	Match(*http.Request) bool
+}
+
+type Scope struct {
+	disabled bool
```

Comment:
Simple reason: bool type is `false` state by default. Also, most common scenario comes first for me, so the most common scenario is to have the plugin enabled, unless you explicitely disable it.